app/serializers/rest/account_serializer.rb

```diff
@@ -6,8 +6,8 @@ class REST::AccountSerializer < ActiveModel::Serializer
 
   # Please update `app/javascript/mastodon/api_types/accounts.ts` when making changes to the attributes
 
-  attributes :id, :username, :acct, :display_name, :locked, :bot, :discoverable, :indexable, :group, :created_at,
-             :note, :url, :uri, :avatar, :avatar_static, :header, :header_static,
+  attributes :id, :username, :acct, :display_name, :locked, :bot, :discoverable, :indexable, :attribution_domains,
+             :group, :created_at, :note, :url, :uri, :avatar, :avatar_static, :header, :header_static,
```

Comment:
Adding this here would mean the API would return attribution domains in a lot of places, as this serializer is quite widely used.

As this PR is about changing the value via the `api/v1/accounts/credentials` controller, how about moving this to the `REST::CredentialAccountSerializer` that is used there and only a handful of other (related) places?